templates/single-meetings.php

```diff
@@ -251,12 +236,36 @@ function add_hour_to_start_time($start) {
 										$location_info .= '<p class="location-region">' . $meeting->region . '</p>';
 									}
 
-									echo tsml_link(
+									/*echo tsml_link(
```

Comment:
Commenting out code? Probably better to just delete and add back later from the archive if required.